### DISCUSSION TITLE
Writing permission on the Wiki

### DISCUSSION THREAD

**Discussion by alaingdl** [gnudatalanguage/gdl#942] [General]
Changing permissions (fine tuning) in Github is not obvious for me. I asked to my student, @eloirozier , now in @gnudatalanguage/gdlteam to make some changes in the Wiki, but he can't, and may be I need to upgrade him
as a member of @gnudatalanguage/gdlpackagers  ? But is it a good idea ? And maybe other members would 
like to write in the Wiki ? How can we do that ? Thanks. A.

**Comment by slayoo** [comment]
Alain, at https://github.com/gnudatalanguage/gdl/settings, Github offers to limit wiki write access to those who have write access to the repo: 
![image](https://user-images.githubusercontent.com/89685/118561691-ac000c00-b76b-11eb-820e-89c21e9efd84.png)
And this seems to be the only way to control it.

**Reply by alaingdl** [reply]
Thanks Sylwester, this is what I understood too, but may be it is not OK to change that ? 

What do you _all_ think on that ? Who want to write in the Wiki too ?

**Reply by slayoo** [reply]
BTW, let's have in mind that wiki contents are now automagically propagated to the website thanks to @pjb7687's script:
https://github.com/gnudatalanguage/gnudatalanguage.github.io/blob/a063236a41ffa7a2bb376a66ed3126f3060c4b95/.github/workflows/deploy.yml#L27

**Reply by olebole** [reply]
I would try out to have it open for all first, as it lowers the threshold to contribute. If we really observe vandalism, we can always revert it, exclude the originators and/or restrict editing.

**Reply by slayoo** [reply]
Is there any way to be notified of Wiki updates?

**Reply by GillesDuvert** [reply]
yes the more the merrier
